<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testimonial</class>
 <widget class="QWidget" name="Testimonial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="write">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit a Testimonial</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="custTest">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="submitForm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Feel Free to Write testimonials below...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
